site/src/pages/TemplateVersionEditorPage/ProvisionerTagsPopover.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { chromatic } from "testHelpers/chromatic";
+import { MockTemplateVersion } from "testHelpers/entities";
+import { ProvisionerTagsPopover } from "./ProvisionerTagsPopover";
+import { useArgs } from "@storybook/preview-api";
+
+const meta: Meta<typeof ProvisionerTagsPopover> = {
+  title: "pages/ProvisionerTagsPopover",
+  parameters: {
+    chromatic,
+    layout: "centered",
+  },
+  component: ProvisionerTagsPopover,
+  args: {
+    tags: MockTemplateVersion.job.tags,
+  },
+  render: function Render(args) {
```

Comment:
Since you are passing the component you don't need this render function at all. At least if you are using it just to test with Chromatic.

Reply:
I wanted this to function in Storybook live, and my understanding was that this was the way to manipulate args getting passed into the component with `useArgs()`.